lang/cs/Org.Apache.REEF.Experimental/ParquetReader/ParquetCollection.cs

```diff
@@ -40,10 +44,18 @@ public class ParquetCollection<T> : IEnumerable<T>, IDisposable
         /// <param name="avroPath">Path to input avro file.</param>
         public ParquetCollection(string avroPath)
         {
-            stream = new FileStream(avroPath, FileMode.Open);
-            avroReader = AvroContainer.CreateReader<T>(stream);
-            seqReader = new SequentialReader<T>(avroReader);
-            _avroPath = avroPath;
+            try
+            {
+                stream = new FileStream(avroPath, FileMode.Open);
+                avroReader = AvroContainer.CreateReader<T>(stream);
+                seqReader = new SequentialReader<T>(avroReader);
+                _avroPath = avroPath;
+            }
+            catch (System.Exception ex)
+            {
+                Logger.Log(Level.Error, ex.Message);
+                throw ex;
```

Comment:
Please throw a specific new exception with a message and don't log it here. Also, please add XMLDocs that describe which exception gets thrown when.